### ISSUE TITLE
Venn diagram with patterns in circle

### ISSUE THREAD

**Issue by kanekalla** [jolars/eulerr#83]
Thanks for a great package @jolars !

I am curious if there are ways to pattern one circle with horizontal or angled lines?
Attached is the representation that I am trying !!

![image](https://user-images.githubusercontent.com/9707250/127686408-b213b908-aaa0-4f99-a006-cc440ee425ec.png)

Thanks !!

**Comment by jolars** [comment]
No, unfortunately not. I don't think patterns are a good idea in general due to Moiré effects, but I'd accept a pull request.

**Comment by jolars** [comment]
Okay, I've implemented this (in the development version) now!

```r
fit <- euler(c(A = 10, B = 8, "A&B" = 3))

plot(
  fit,
  fills = list(fill = c("#F8766D", "#00BFC4")),
  patterns = list(
    type = c("stripes", "stripes"),
    angle = c(0, 90),
    alpha = 1,
    col = "black",
    lwd = 0.5
  ),
  legend = TRUE,
  quantities = FALSE
)
```

<img width="673" height="666" alt="Image" src="https://github.com/user-attachments/assets/93bf374e-43a3-45f6-8c2a-c7692b5c51d4" />
